src/components/CurrentUser/CurrentUser.jsx

```diff
@@ -1,12 +1,62 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+
 import './CurrentUser.scss';
+import { getUser } from '../../api/api';
+
+export class CurrentUser extends React.Component {
+  state = {
+    currentId: 0,
+    user: {},
+  }
+
+  loadUser = async() => {
+    const user = await getUser(this.props.userId);
+
+    if (!user.data) {
+      this.props.clearUser();
```

Comment:
```suggestion
      return;
```
навіщо тут викликати clear user? якщо данних там немає, то можна просто завершувати функцію

Reply:
Ну просто щоб не склалось хибне враження, що виведена інфа - це інфа про юзера, якого немає. Тому я і очищаю повністю інфу